.github/workflows/formatter.yaml

```diff
@@ -2,15 +2,14 @@ name: formatter
 on: [pull_request]
 jobs:
   formatter:
+    name: Runic formatting
     runs-on: ubuntu-latest
     steps:
-      - name: checkout repo
-        uses: actions/checkout@v4
-      - name: install Julia
-        uses: julia-actions/setup-julia@v1
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
         with:
-          version: "1.10"
-      - name: Install Julia requirements
-        run: julia --project=${GITHUB_WORKSPACE}/.formatting -e 'import Pkg; Pkg.instantiate()'
-      - name: Check code style
-        run: julia --project=${GITHUB_WORKSPACE}/.formatting ${GITHUB_WORKSPACE}/.formatting/format_all.jl
+          version: "1.11"
```

Comment:
I think we should either use the stable version, i.e. 1.10, or just the latest, i.e. '1'.
If we use 1.11, we probably won't remember to update it to every new minor release in each of our packages.